src/lightning_app/testing/testing.py

```diff
@@ -127,6 +128,33 @@ def browser_context_args(browser_context_args: Dict) -> Dict:
     }
 
 
+@contextmanager
+def run_cli(args) -> Generator:
+    """This utility is used to automate end-to-end testing of the Lightning AI CLI."""
+    env_copy = os.environ.copy()
+    env_copy["PREPARE_LIGHTING"] = "1"
```

Comment:
I don't think you need `PREPARE_LIGHTING`.

Reply:
This is meant only to package the lightning codebase.

Reply:
gotcha, I'll remove the environment variable because this is not a lightning app.